### PR TITLE
Hot reloading extravaganza

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers 

:fire: Hot reloading in the desktop client using react-hot-loader 3.0.0 beta
:fire: Live local-debug filter for twiddling the dumb sheet from your editor
:sparkle: Cleaned up desktop entry point setup
:one: Moved `bootstrap` action out of component so it only runs on startup 
:left_right_arrow: Unified live local debug properties into a single file

Deps: https://github.com/keybase/js-vendor-desktop/pull/3

:fire:

### CODE REVIEW

File: desktop/npm-helper.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ const commands = {
   },
   'hot-server': {
     env: {HOT: 'true', DEBUG: 'express:*'},
-    nodeEnv: 'production',
+    nodeEnv: 'development',
```
Comment (chromakode) [inline desktop/npm-helper.js]: Necessary for react-hot-loader to perform its source transform.

File: desktop/renderer/index.js [hunk 2]
```diff
@@ -3,131 +3,107 @@
...
 import electron, {ipcRenderer} from 'electron'
-import RemoteManager from './remote-manager'
 import {ipcLogsRenderer} from '../app/console-helper'
 import loadPerf from '../shared/util/load-perf'
 import merge from 'lodash/merge'
-import {MuiThemeProvider} from 'material-ui/styles'
-import materialTheme from '../shared/styles/material-theme.desktop'
 
-ipcLogsRenderer()
+function setupApp (store) {
```
Comment (chromakode) [inline desktop/renderer/index.js]: It's easier to read this diff if you ignore whitespace. You can do that by adding `?w=1` to the URL.

File: desktop/renderer/container.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    super()
+
+    this.state = {
+      panelShowing: false,
+    }
+
+    if (__DEV__) { // eslint-disable-line no-undef
+      window.addEventListener('keydown', event => {
+        if (event.ctrlKey && event.keyCode === 72) {
+          this.setState({panelShowing: !this.state.panelShowing})
```
Comment (MarcoPolo) [inline desktop/renderer/container.js]: I think we can kill this panelShowing code.

**Comment by MarcoPolo** [comment]
👍 🎉 🎆

File: desktop/renderer/container.js (hunk 3 above)
Comment (chromakode) [inline desktop/renderer/container.js]: Ah, duh! Doesn't seem to be doing anything atm :)

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**